testing/examples/frame_system_tests.rs

```diff
@@ -48,6 +48,10 @@ async fn main() {
 	// GetAccountInformation
 	let _account_info = api.get_account_info(&alice).unwrap().unwrap();
 	let _account_data = api.get_account_data(&alice).unwrap().unwrap();
+	// Empty account information
+	let ferdie = AccountKeyring::Two.to_account_id();
+	assert!(api.get_account_info(&ferdie).unwrap().is_none());
```

Comment:
If you're doing this in one line, it might be confusing to discern what has failed (respective on which line).
It think is more intuitive if you make a two liner out of this:
```rust
let maybe_account_info = get_account_info(&ferdie).unwrap();
assert!(maybe_account_info.is_none())
```

Reply:
What about `let _account_data = api.get_account_data(&alice).unwrap().unwrap();` ? also put on 2 lines to know which unwrap failed?

Reply:
would be better yeah, but then we'd have to make a whole lot changes... (on the other files) and - so no, leave as is.